### DISCUSSION TITLE
Как делать сервис?

### DISCUSSION THREAD

**Discussion by marshelx** [bol-van/zapret#944] [Q&A]
Я прочитал, то что нужно редактировать программы service_create.cmd и т.д. Но как их редактировать или что с ними сделать я не знаю, можете помочь?

**Comment by VsevkaD** [comment]
Немного поздно, на будущее для тех кто будет искать.
Для того чтобы сервис работал как preset_russia.cmd

1. В файле **preset_russia.cmd**  Скопируй всё начиная со второй строчки!
2. В файле **service_create.cmd** после строчки **set ARGS=^** замени все строчки до **rem call :srvinst winws1**
3.  В строчке **rem call :srvinst winws1** удали **rem** чтобы получилось **call :srvinst winws1**
4. Сохрани и запусти **service_create.cmd** от имени администратора, в службах появиться **zapret DPI bypass software : winws1**, все должно работать, возможно придется подождать секунд 10, или перезагрузить браузер/дискорд/пк.

В других файлах ничего делать не надо, если нужно остановить/запустить/удалить эту службу, запускай соответствующие файлы от имени администратора.
